vue/src/components/ModeSelector.vue

```diff
@@ -16,15 +16,15 @@ const proposalView = computed(() => route.path.includes('proposals'));
   >
     <span>Mode:</span>
     <v-btn
-      to="/"
+      :to="scoringApp ? '/scoring/' : '/'"
       :color="!proposalView? 'primary': ''"
       :theme="!proposalView? 'dark': ''"
       class="mx-2"
     >
       Analyst
     </v-btn>
     <v-btn
-      to="/proposals"
+      :to="scoringApp ? '/scoring/proposals' : '/proposals'"
```

Comment:
Isn't this just re-implementing `ApiService.getApiPrefix()`?
```suggestion
      :to="`${ApiService.getApiPrefix()}/proposals`"
```

Reply:
Yes, I think it's a habit from my other projects.  I utilize `defineComponent` instead of `<script setup>` that way I can be explicit with the `setup()` functions return values.  It just doesn't feel right to me to provide a whole Static Class like `ApiService` to the template so I typically change it to have a property I'm looking for in a simple computed value and return that.

There's also the fact that I'm not importing or using `ApiService` at all in this component.  I'm utilizing the `route` because we don't use `ApiService.getPrefix()` with `/proposals`.